src/Command/DoctrineCommand.php

```diff
@@ -59,6 +61,8 @@ protected function getEntityManager($name, $shardId = null)
             throw new InvalidArgumentException('Shards are not supported anymore using doctrine/dbal >= 3');
         }
 
+        assert($manager instanceof EntityManager);
```

Comment:
I think this assertion will cause issue for projects decorating the EntityManager. We should probably use EntityManagerInterface as type instead.

Reply:
Thanks for feedback. Addressed in https://github.com/doctrine/DoctrineBundle/commit/8ab3cb317ac4d7f218095d54cceb14167b2e26e6